config/sources/common.conf

```diff
@@ -6,19 +6,180 @@
 # This file is a part of the Armbian Build Framework
 # https://github.com/armbian/build/
 #
+
 # this is sourced before any other arch specific config file, always. see main-config.sh
 
+# Source the mainline kernel hooks, in a separate file for easier maintenance.
```

Comment:
"Mainline" is already getting code on common ground. Terminology `common` and `mainline` could perhaps be one?

Reply:
No. `common` is for _all_ kernels (including 3rd party / vendor / etc), while `mainline` is for strictly mainline kernels, subject to the auto-bump / lock / etc rules proposed here.

Reply:
Yes. What I meant to hold the mainline part of the code in common. But this is just a comment, both ways play.